db/migrate/20160405011331_fill_in_orderings.rb

```diff
@@ -0,0 +1,18 @@
+class FillInOrderings < ActiveRecord::Migration
```

Comment:
this should be a part of the original adding ordering migration.

additionally, we need to write this in SQL. here's [two](https://github.com/YaleSTC/reservations/blob/master/db/migrate/20160304025530_add_overdue_count_to_equipment_models.rb) [examples](https://github.com/YaleSTC/reservations/blob/master/db/migrate/20150405012126_remove_approval_status_from_reservation.rb) of other migrations in our codebase where we had to do this.